nodes/test/audio_tts/conftest.py

```
@@ -0,0 +1,109 @@
+    try:
+        import numpy as np
+
+        return hasattr(np, 'testing')
+    except Exception:
+        return False
+
+
+HAS_NUMPY = _has_real_numpy()
+
+
+def pytest_collection_modifyitems(config, items):
```

Ruff (0.15.9)

[warning] Unused function argument: `config`

(ARG001)